README.md

```diff
@@ -18,27 +29,29 @@ This package contains transformation models, designed to work simultaneously wit
 | [marketo__leads](https://github.com/fivetran/dbt_marketo/blob/main/models/marketo__leads.sql)           | Each record represents a Marketo lead, enriched with metrics about email performance.                                                          |
 | [marketo__programs](https://github.com/fivetran/dbt_marketo/blob/main/models/marketo__programs.sql)         | Each record represents a Marketo program, enriched with metrics about email performance.                                                       |
 
+# 🎯 How do I use the dbt package?
+
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
 
-## Installation Instructions
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+- At least one Fivetran Marketo connector syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, or **Databricks** destination.
 
-Include in your `packages.yml`
+## Step 2: Install the package
+Include the following Marketo package version in your `packages.yml` file.
 
-```yaml
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+
+```yml
 packages:
   - package: fivetran/marketo
-    version: [">=0.7.0", "<0.8.0"]
+    version: [">=0.8.0", "<0.9.0"]
 ```
 
-## Configuration
-By default, this package will look for your Marketo data in the `marketo` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Marketo data is , please add the following configuration to your `dbt_project.yml` file:
+## Step 3: Define database and schema variables
+By default, this package runs using your destination and the `marketo` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Marketo data is (for example, if your Marketo schema is named `marketo_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
   marketo_source:
```

Comment:
Let's remove the marketo_source piece and just have the variables be declared globally to be consistent with our other packages and help avoid confusing the users.

Reply:
Done.